docs/recommendations/recommendations.md

```diff
@@ -37,13 +37,13 @@ getRecs({user, unixTimestamp, signature, recommendation, recommendations}: Recom
 ```
 
 ##### RecommendationQuery
-| Name                                     | Description                                                                                                        | Type                                                                                                 |
-|------------------------------------------|--------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
-| **user**                                 | Information about the user                                                                                         | [user](https://docs.breinify.com/?javascript--node#requesting-a-product-recommendation)              |
-| **unixTimestamp**                        | The number of seconds that have elapsed since 00:00:00 Coordinated Universal Time (UTC), Thursday, 1 January 1970. | `number`                                                                                             |
-| **signature**                            | Signature to authenticate a request, is only available if the the secret for the Breinify API key is configured.   | `string`                                                                                             |
-| **recommendation**<br/>***required*****  | Parameters set for making the recommendations.                                                                     | [recommendation](https://docs.breinify.com/?javascript--node#requesting-a-product-recommendation)    |
-| **recommendations**<br/>***required***** | A list of `recommendation`                                                                                         | [[recommendations]](https://docs.breinify.com/?javascript--node#requesting-a-product-recommendation) |
+| Name                                     | Description                                                                                                        | Type                                                                                                       |
+|------------------------------------------|--------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| **user**                                 | Information about the user                                                                                         | [user](https://docs.breinify.com/?javascript--node#requesting-a-product-recommendation)                    |
+| **unixTimestamp**                        | The number of seconds that have elapsed since 00:00:00 Coordinated Universal Time (UTC), Thursday, 1 January 1970. | number                                                                                                     |
```

Comment:
I'm not sure if we need to define what a unixTimestamp is to the user, but rather, we should probably provide a description of what the unixTimestamp will be used for in terms of the recommendation query

Reply:
good point, i was following the old docs, will see if there's a better definition